app/routes/Password/containers/Password.js

```diff
@@ -42,26 +42,18 @@ class Password extends Component {
   }
 
   render() {
-    const { password, errorMessage } = this.state
-    const { dest, option } = this.properties
+    const {
+      password,
+      errorMessage
+    } = this.state
 
-    let state
     if (this.state.continue) {
-      if (option === 'updatePassword') {
-        state = {
-          option,
-          oldPassword: password,
-        }
-      }
-      else {
-        state = {
-          option,
+      return <Redirect to={{
+        pathname: this.properties.dest,
+        state: {
+          ...this.properties,
```

Comment:
I do not know if ```this.properties``` has complex attributes, but if it has, you can have mutations problems because ```...``` as far as I remember do not create complex structures, just reference them.